src/common/darktable.h

```diff
@@ -480,6 +512,11 @@ static inline size_t dt_get_total_memory()
   size_t length = sizeof(uint64_t);
   sysctl(mib, 2, (void *)&physical_memory, &length, (void *)NULL, 0);
   return physical_memory / 1024;
+#elif defined _WIN32
+  MEMORYSTATUSEX memInfo;
+  memInfo.dwLength = sizeof(MEMORYSTATUSEX);
+  GlobalMemoryStatusEx(&memInfo);
+  return memInfo.ullTotalPhys / 1024;
```

Comment:
i may be wrong, but it might be better to add some suffixes to `1024` here, else maybe the wrong type will be used?
Or explicitly cast both operands to some big type.